agent_testing/agent.py

```diff
@@ -118,9 +118,13 @@ async def return_list(self, message, conn):
     # Delete string from the list
     @route("https://example.com/manage-list/0.1/delete")
     async def delete_item(self, message, conn):
-        print("Deleting string of index", message["item"], "from the list.")
         self.strings_list.remove(self.strings_list[message["item"]])
-        print("Updated list: ", self.strings_list)
+        await conn.send_async(
+            {
+                "@type": "https://example.com/manage-list/0.1/delete",
```

Comment:
The message that comes into this handler is `https://example.com/manage-list/0.1/delete`. This message acts as our "request" so we should probably have a different message type for the "response". Perhaps `https://example.com/manage-list/0.1/deleted`?